setup.py

```diff
@@ -60,6 +60,24 @@
 if sys.platform == "darwin":  # osx
     ext_compiler_args.append("-mmacosx-version-min=10.9")
 
+ext_list = [
+    Extension(
+        "histomicstk.segmentation.label.trace_boundaries_cython",
+        sources=["histomicstk/segmentation/label/trace_boundaries_cython.pyx",
+                 "histomicstk/segmentation/label/trace_boundaries_opt.cpp"],
+        include_dirs=[numpy.get_include()],
+        extra_compile_args=ext_compiler_args,
+        language="c++",
+    ),
+
+    Extension(
+        "histomicstk.segmentation.nuclear._max_clustering_cython",
+        sources=["histomicstk/segmentation/nuclear/_max_clustering_cython.pyx"],
+        include_dirs=[numpy.get_include()],
+        extra_compile_args=ext_compiler_args,
+        language="c++",
```

Comment:
Seems like that _max_clustering_cython tends to use c rather than c++. So, you may want to delete this.